variables.tf

```diff
@@ -18,11 +13,6 @@ variable "tls_key" {
   description = "The private key corresponding to the signed certificate tls_cert."
 }
 
-variable "base_domain" {
-  type        = string
-  description = "<var.deployment_id>.<var.route53_domain>"
-}
-
```

Comment:
Will these removed variables be hard-coded in our `astronomer_helm_values` variable?

Reply:
Nope, they will still be in TF variable but in the other TF module -> https://github.com/astronomer/terraform-google-astronomer-cloud/pull/8/files